packages/core/src/tracing/cls.js

```diff
@@ -349,17 +349,19 @@ function setCurrentSpan(span) {
 
 /**
  * Get the currently active span.
+ * @param {boolean} [fallbackToSharedContext=false]
  * @returns {InstanaSpan}
  */
-function getCurrentSpan() {
-  return ns.get(currentSpanKey);
+function getCurrentSpan(fallbackToSharedContext = false) {
+  return ns.get(currentSpanKey, fallbackToSharedContext);
 }
 
-/*
+/** *
```

Comment:
_nitpick_: this looks odd, probably a typo
```suggestion
/**
```